templates/utils.html

```diff
@@ -104,6 +110,10 @@
 					<img loading="lazy" alt="Post image" src="{{ post.media.url }}"/>
 				</desc>
 		</svg>
+		{% else %}
+		<!-- debug: simplified image thumbnail -->
+		<img loading="lazy" alt="Post image" src="{{ post.media.url }}"/>
```

Comment:
Ah, I think we might have to set the width/height here explicitly in the path where we have non-zero values?

It hard for me to test content-reflow. If anyone knows how to do it reliably, let me know please.

Reply:
Yeah, if we have non-zero width and height, we should definitely use them.

To test content-reflow, I recommend refreshing with Control + F5, which clears cache so the image might take longer to load and you can see the split-second where the comments jump from the middle of the page to the bottom